config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
+
+function parseBoolean(boolToParse) {
+  if (boolToParse === 'true') return true;
+  if (boolToParse === 'false') return false;
+  return undefined;
+}
+
+module.exports = {
+  baseUrl: process.env.COASTER_BASE_URL,
+  withAcl: parseBoolean(process.env.COASTER_ACL),
+  language: process.env.COASTER_DEFAULT_LANGUAGE,
```

Comment:
Ah, this will not work in the browser without dotenv-webpack, i may consider changing this to where the hardcoding happens rather than pulling in env vars at runtime. Implementing dotenv-webpack can be a future bit of work if it is deemed of value, the value to the project at this stage is very low though.